### DISCUSSION TITLE
Official Guidance on using `next-urql` with a custom `_app` component in Next.js

### DISCUSSION THREAD

**Discussion by h-jennings** [urql-graphql/urql#2215] [Q&A] [answered]
First off thank you so much for this excellent library. It's really fantastic.

I'm wondering if there's official guidance on **how to properly do SSR if you have a custom `_app` component that's calling `getInitialProps`.** The official examples for using `withUrqlClient` on the `App` component-level don't call `getInitialProps`. It appears that others also are looking for some insight here since I seen and read other issue/discussion threads on the topic.

**Comment by JoviDeCroock** [comment, marked as answer]
Hey, it's just like the Next docs suggest made a [small example](https://codesandbox.io/s/laughing-kilby-7i165?file=/pages/_app.js)

**Reply by JoviDeCroock** [reply]
😅 it was earlier, might've made a mistake, should be able to log it out

Edit: seems to work for me ![Screenshot of app showing injected prop](https://user-images.githubusercontent.com/17125876/150790728-0a85b459-1b41-409a-b397-4b109d5347b5.jpeg) next sometimes acts up

**Reply by h-jennings** [reply]
Classic typo! Thanks for the helpful reply!
